vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/MaxInFlightByWorkflow.java

```diff
@@ -75,6 +76,22 @@ public ConsumableResourceResponse request(
     }
   }
 
+  /**
```

Comment:
I've been keeping methods in alphabetical order. If you're IntelliJ, _Code_ -> _Rerrange Code_ should do it.